source/fluid/backend/package.d

```diff
@@ -976,6 +979,10 @@ struct Texture {
         && dpiX == other.dpiX
         && dpiY == other.dpiY;
 
+    version (Have_raylib_d)void opAssign(raylib.Texture rayTexture) @system {
+        this = (cast(Raylib5Backend)defaultFluidBackend).toFluid(rayTexture, Image.Format.rgba);
```

Comment:
You're using the wrong function here, it should be the global one, not the private backend one.